<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addDialog</class>
 <widget class="QDialog" name="addDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>225</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавление записи</string>
  </property>
  <widget class="QLabel" name="mileageLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Пробег</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="yearInput1">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1886</number>
   </property>
   <property name="maximum">
    <number>3000</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="colorInput">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="accessibleDescription">
    <string/>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Введите цвет</string>
   </property>
  </widget>
  <widget class="QLabel" name="modelLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Марка</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="modelInput">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="accessibleDescription">
    <string/>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Введите марку</string>
   </property>
  </widget>
  <widget class="QLabel" name="yearLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Год выпуска</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="priceInput1">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>99999999</number>
   </property>
   <property name="singleStep">
    <number>100000</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="changeButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>190</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QLabel" name="colorLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Цвет</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="priceLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Начальная цена</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="mileageInput1">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>999999</number>
   </property>
   <property name="singleStep">
    <number>50000</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="manufacturerInput">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Введите производителя</string>
   </property>
  </widget>
  <widget class="QLabel" name="manufacturerLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Производитель</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>manufacturerInput</tabstop>
  <tabstop>modelInput</tabstop>
  <tabstop>yearInput1</tabstop>
  <tabstop>colorInput</tabstop>
  <tabstop>mileageInput1</tabstop>
  <tabstop>priceInput1</tabstop>
  <tabstop>changeButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
